shell/common/shell.cc

```diff
@@ -385,9 +398,6 @@ void Shell::RunEngine(RunConfiguration run_configuration,
   FML_DCHECK(is_setup_);
   FML_DCHECK(task_runners_.GetPlatformTaskRunner()->RunsTasksOnCurrentThread());
 
-  if (!weak_engine_) {
```

Comment:
Why is this getting removed?

Reply:
we can't deref weak_engine_ in this thread and we do the same check inside the UITaskRunner lambda immediately after this anyway.

Reply:
Ahh I see, the git diff was collapsing things right before the good parts.